### PR TITLE
Feature/update landing page

### PR DESCRIPTION
**Pull request by kbgar24**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +36,7 @@ No generators installed (yet). Installing one will populate this section in the
 ## Made with Concatapult
 
 See [the `pult` docs](https://github.com/Concatapult/pult#readme) for adding additional modules to this boilerplate.
+
```
Comment (shamsup) [inline README.md]: In the future, leave out a change like this.

File: README.md [hunk 2]
```diff
@@ -36,3 +36,7 @@ No generators installed (yet). Installing one will populate this section in the
...
+
+
+
```
Comment (shamsup) [inline README.md]: in the future, leave out a change like this.

**Comment by shamsup** [comment]
And aren't we trying to use sass with materialize? Why is it all set up to be static? Additionally, what advantage does using the `materialize-css` module give us over just downloading their sass source code and adding it into our client directory?

**Comment by kbgar24** [comment]
For dev time's sake, I think it prudent to continue with static CSS for the interim, at least for my contributions. Implementing Redux has proven to exhibit a sufficiently steep learning curve, and will be my primary focus. Will look to refactor to SASS after MVP is achieved.

File: client/public/index.html [hunk 3]
```diff
@@ -14,7 +14,7 @@
 	  <link href="./styles/nav-bar.css" rel="stylesheet"></link>
 	  <link href="./styles/splash-container.css" rel="stylesheet"></link>
 	  <link href="./styles/features.css" rel="stylesheet"></link>
-	  <link href="../lib/prism/prism.css" rel="stylesheet"></link>
+	  <link href="./styles/product-select.css" rel="stylesheet"></link>	  
 	  <link href="../../node_modules/materialize-css/dist/materialize.min.css" rel="stylesheet"></link>
```
Comment (shamsup) [inline client/public/index.html]: We can't include styles from `node_modules` as they aren't available via routing. This is intentional.

File: client/public/index.html [hunk 4]
```diff
@@ -28,7 +28,7 @@
 	  <script src="js/materialize.min.js"></script>
 	  <script src="js/init.js"></script>
 		<script src="../lib/prism/prism.js"></script>
```
Comment (shamsup) [inline client/public/index.html]: With no prism, this needs to be removed.

**Comment by shamsup** [comment]
Please rebase instead of pull. There should be no merge commits in pull requests.

**Comment by shamsup** [comment]
In the future, run your linting before a pull request as well. There are multiple files where tabs are being used instead of spaces.

**Pull request merged**